src/api/java/com/minecolonies/api/blocks/AbstractBlockHut.java

```diff
@@ -103,7 +108,7 @@ public float getDestroyProgress(final BlockState state, @NotNull final PlayerEnt
         if (building != null && !building.getChildren().isEmpty() && (player.level.getGameTime() - lastBreakTickWarn) < 100)
         {
             lastBreakTickWarn = player.level.getGameTime();
-            LanguageHandler.sendPlayerMessage(player, "block.minecolonies.blockhut.breakwarn.children");
+            MessageUtils.format(HUT_BREAK_WARNING_CHILD_BUILDINGS).sendTo(player);
```

Comment:
Seems weird calling it format if we give it the main string. MessageUtils.message().append().sendTo() feels better

Reply:
it's `format(id, args...)`, following in the fine tradition of `string.format` in various languages, albeit with a translation key instead of a literal format string.  `translate(...)` could be another reasonable method name; `message(...)` is probably fine too, but sounds a little redundant when on `MessageUtils`.

Reply:
So do I change this to `translate` instead? Don't want to have to rename it all again unless it's for a good reason